docs/architecture/adr-template.md

```diff
@@ -8,13 +8,47 @@
 
 > This section contains all the context one needs to understand the current state, and why there is a problem. It should be as succinct as possible and introduce the high level idea behind the solution.
 
+## Alternative Approaches
+
+> This section contains information around alternative options that are considered before making a decision. It should contain a explanation on why the alternative approach(es) were not chosen.
+
 ## Decision
 
 > This section explains all of the details of the proposed solution, including implementation details.
 > It should also describe affects / corollary items that may need to be changed as a part of this.
 > If the proposed change will be large, please also indicate a way to do the change to maximize ease of review.
 > (e.g. the optimal split of things to do between separate PR's)
```

Comment:
This should be removed now right - it should be covered in the detail design?

Reply:
The wording in Decision should change but I feel the decision and why it was chosen should go here and then the detailed design expands on the impl.

Reply:
Yeah sorry for the confusion I just meant the last two bullet points

Reply:
oh okay. thanks for clarifying